api/maps/MarkerInfoWindow.tsx

```diff
@@ -1,23 +1,44 @@
 import { useCallback, useEffect, useState } from 'react';
+import Image from 'next/image';
 import { MarkerClusterer } from '@googlemaps/markerclusterer';
 import {
   AdvancedMarker,
   InfoWindow,
+  Pin,
   useAdvancedMarkerRef,
 } from '@vis.gl/react-google-maps';
+import energyStorage from '../../assets/Custom-Markers/energy_storage.svg';
+import geothermal from '../../assets/Custom-Markers/geothermal.svg';
+import hydroelectric from '../../assets/Custom-Markers/hydroelectric.svg';
+import landbased_wind from '../../assets/Custom-Markers/landbased_wind.svg';
+import offshore_wind from '../../assets/Custom-Markers/offshore_wind.svg';
+import pumped_storage from '../../assets/Custom-Markers/pumped_storage.svg';
+import solarPower from '../../assets/Custom-Markers/solar_power.svg';
+
+const technologyToPin: Record<string, string> = {
+  'Energy Storage': energyStorage,
+  Geothermal: geothermal,
```

Comment:
Why do some have the quotations marks `'' `and others don't?

Reply:
@itsliterallymonique I have no clue... I had the quotes around them but I think prettier takes them away for some of them for some reason if the keys are exact matches to the values (e.g. they have no extra spaces or anything)

Reply:
oh okay got it. Thanks deena!